openpype/hosts/blender/plugins/publish/increment_workfile_version.py

```diff
@@ -9,7 +9,7 @@ class IncrementWorkfileVersion(pyblish.api.ContextPlugin):
     label = "Increment Workfile Version"
     optional = True
     hosts = ["blender"]
-    families = ["animation", "model", "rig", "action", "layout"]
+    families = ["workfile"]
```

Comment:
why don't you simply add `workfile` to the list?

Reply:
I don't understand why there are other families than workfile on this plugin. If it's about incrementing workfile version number, what do the other families have to do with it?

Reply:
The main idea is to always be able to retrieve the file which has produced a representation. Therefore, if you publish anything, you shouldn't be able to edit the source file after, but only an incremented copy.

The issue with OP is they allow to not publish a workfile with the rest, which is not a good practice, but currently this is how it works.

Reply:
Okay, I understand now, thanks you. I readded the families I removed.